.ado/templates/prep-and-pack-nuget.yml

```diff
@@ -34,6 +35,14 @@ steps:
         slices:  ${{ parameters.slices }}
         buildProperties: CommitId=${{parameters.publishCommitId}};nugetroot=${{parameters.nugetroot}};baseconfiguration=Release;baseplatform=x64
 
+  - ${{ if eq(parameters.packMicrosoftReactNativeCxx, true) }}:
```

Comment:
OOC, do we ever see us disabling this?

Reply:
No, the original nuget code had flags for each package, so I just kept them. I can remove them in a future opr